src/Validation/ValidationServiceProvider.php

```diff
@@ -57,7 +65,7 @@ private function returnForbidden(): string
 
         status_header(403);
 
-        add_filter('pre_get_document_title', function (string $title) {
+        add_filter('pre_get_document_title', function ($title) {
```

Comment:
This can be `null`.

Reply:
`string $title = ''` zou ook kunnen volgens mij.

Reply:
Of `?string`